src/components/cloud-cover/cloud-cover.twig

```diff
@@ -1,12 +1,30 @@
+{% set _heading_block %}{% block heading %}{% endblock %}{% endset %}
+{% set _content_block %}{% block content %}{% endblock %}{% endset %}
 {% set _extra_block %}{% block extra %}{% endblock %}{% endset %}
 {% set _scene_block %}{% block scene %}{% endblock %}{% endset %}
 
 <div class="c-cloud-cover t-dark{% if class %} {{ class }}{% endif %}">
   <div class="o-container o-container--pad-inline">
     <div class="o-container__content c-cloud-cover__inner">
       <div class="c-cloud-cover__content">
-        {% block content %}
-        {% endblock %}
+        {% embed '@cloudfour/objects/rhythm/rhythm.twig' with {
+          class: 'o-rhythm--condensed',
+          _heading_block: _heading_block,
+          _content_block: _content_block
+        } only %}
+          {% block content %}
+            <div class="c-cloud-cover__heading">
+             {{ _heading_block }}
+            </div>
+            <div class="c-cloud-cover__copy">
+              {% embed '@cloudfour/objects/rhythm/rhythm.twig' with {} %}
```

Comment:
You are probably going to want this to be `with { _content_block: _content_block } only` in order to consistently have access to `_content_block` without inheriting other values.